src/http/ngx_http_request.c

```diff
@@ -1247,9 +1243,15 @@ ngx_http_process_request_line(ngx_event_t *rev)
                 }
 
                 r->connect_port_n = port;
-            }
+
+                /* skip processing request uri */
+            } else
 #endif
 
+            if (ngx_http_process_request_uri(r) != NGX_OK) {
+                break;
+            }
+
```

Comment:
try to fixed bug: https://github.com/chobits/ngx_http_proxy_connect_module/pull/248